CHANGELOG.md

```diff
@@ -23,6 +23,26 @@
 9. [Continuous integration](https://travis-ci.org/github/idontgetoutmuch/random).
 10. Fully documented - for more details see the [haddock](https://htmlpreview.github.io/?https://github.com/idontgetoutmuch/random/blob/release-notes/docs/System-Random.html).
 
+### Issues Addressed
+
+ Issue Number | Description | Comment
+--------------|-------------|--------
+ [25](https://github.com/haskell/random/issues/25) | The seeds generated by split are not independent | Fixed: changed algorithm to SplitMix, which provides a robust split operation
+ [26](https://github.com/haskell/random/issues/26) | Add Random instances for tuples | Addressed: added `Uniform` instances for up to 6-tuples
+ [44](https://github.com/haskell/random/issues/44) | Add Random instance for Natural | Addressed: added UniformRange instance for Natural
+ [51](https://github.com/haskell/random/issues/51) | Very low throughput | Fixed: see benchmarks below
+ [53](https://github.com/haskell/random/issues/53) | incorrect distribution of randomR for floating-point numbers | (*)
```

Comment:
```suggestion
 [53](https://github.com/haskell/random/issues/53) | incorrect distribution of randomR for floating-point numbers | (\*)
```

`*` starts italic formatting. A backslash escapes it, so editors don't get confused: https://daringfireball.net/projects/markdown/syntax#backslash